tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
add comments for anything non standard / non obvious to us typescript noobs please

Reply:
it has been annotated!